### DISCUSSION TITLE
Send single E-Mails to Subscribers/Users/E-Mail-Addresses

### DISCUSSION THREAD

**Discussion by konsti2020** [hizzle-co/noptin#521] [Ideas]
Hello Noptin-Team,
with the Noptin Newsletter and the Noptin E-Mail Template most of my needs are perfectly covered
but sometimes I need to write single E-Mails to new Customers or Subscribers.
Is there a option to do this? Because obviously it would be nice to have the same good looking E-Mails not only for WordPress Stuff,
like Newsletters and Account Stuff, but also for direct Contact with Single Subscribers, Users and potential new Customers.

Best Regards
 
Konsti2020

**Comment by picocodes** [comment]
We'll soon work on this. It's a great idea.

**Comment by picocodes** [comment]
Hi @konsti2020,

I've pushed a commit that references this discussion:- https://github.com/hizzle-co/noptin/commit/261d8c8c1b44e795e053c93e04ee716b38a4d006

At the moment, we'll only add a "Send Email" button at the top of a subscriber's overview page. In the future, we'll also add a bulk action to select subscribers in bulk and then send them an email.

**Reply by picocodes** [reply]
We'll add the same to users and customers via the Ultimate Add-ons Pack.

**Comment by konsti2020** [comment]
That sounds perfect.
I'm looking forward to try it.
